tools/m2ctx.py

```diff
@@ -7,6 +7,46 @@
 root_dir = script_dir + "/../"
 src_dir = root_dir + "src/"
 
+# Read through the processes context and replace whatever
+def replace_things(output):
```

Comment:
I prefer if the name of this function would be a bit less generic. Something like `custom_replacements` for example?